### PR TITLE
Specify machine queue to use for Windows CI runs.

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
* Broken by buildtools change: https://github.com/dotnet/buildtools/pull/2152
* Ensuring we run on machines with at lease VS 15.7 installed should fix the problem.

### CODE REVIEW

**Comment by StephenBonikowsky** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -220,10 +218,12 @@ def supportedFullCycleOuterloopPlatforms = ['Windows_NT', 'Ubuntu14.04', 'Ubuntu
             }
 
             // Set the affinity.  OS name matches the machine affinity.
-            if (os == 'Windows_NT' || os == 'OSX10.12') {
-                // Set affinity for elevated machines on Windows
-                Utilities.setMachineAffinity(newJob, os, 'latest-or-auto-elevated')
+            if (os == 'Windows_NT') {
```
[the remark this answers is not in the document]
Comment (StephenBonikowsky) [reply netci.groovy]: Ah, lol, right, thanks.

**Comment by adiaaida** [comment]
Looks good to me!

**Comment by StephenBonikowsky** [comment]
Thank you @adiaaida

**Pull request merged**